manifests/03-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    config.openshift.io/inject-proxy: cloud-credential-operator
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+  name: cloud-credential-operator
+  namespace: openshift-cloud-credential-operator
+spec:
+  replicas: 1
+  revisionHistoryLimit: 4
+  selector:
+    matchLabels:
+      app: cloud-credential-operator
+  template:
+    metadata:
+      labels:
+        app: cloud-credential-operator
+    spec:
+      containers:
+      - args:
+        - |
+          if [ -s /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem ]; then
+              echo "Copying system trust bundle"
+              cp -f /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+          fi
+          exec /usr/bin/cloud-credential-operator operator --log-level=debug
+        command:
+        - /bin/bash
+        - -ec
+        env:
+        - name: RELEASE_VERSION
+          value: 0.0.1-snapshot
+        - name: AWS_POD_IDENTITY_WEBHOOK_IMAGE
+          value: quay.io/openshift/aws-pod-identity-webhook:latest
```

Comment:
Is this the value that gets used? Is it acceptable for us to have this kind of implied image which is not reflected in the release image itself? Is the use of latest permanent as well?

Reply:
`quay.io/openshift/aws-pod-identity-webhook:latest` is a placeholder that is replaced by the CVO with the image pull spec during release image creation
https://github.com/openshift/cloud-credential-operator/blob/master/manifests/image-references#L9-L12